tools/gui/qtterminsuche.py

```diff
@@ -50,6 +52,14 @@ def __init__(self, kontaktdaten: dict, zeitrahmen: dict, ROOT_PATH: str, check_d
         self.ROOT_PATH = ROOT_PATH
         self.check_delay = check_delay
 
+        self.use_telegram=False
+        telegram_data=load_telegram_data("data/telegram.json")
```

Comment:
Funktioniert das so mit dem Pfad?
Denn wir sind hier ja in `./tools/gui`

```suggestion
        telegram_data=load_telegram_data(os.path.join(ROOT_PATH, "data", "telegram.json"))
```

Reply:
Leider kann ich die gui version nicht testen weil ich den fehler bekome:
```
  File "xxxxxxxx\xxxx\tools\utils.py", line 158, in update_available
    if latest_version.strip() == current_version.strip():
AttributeError: 'NoneType' object has no attribute 'strip'
```
Aber wie du es vorgeschlagen hattest sollte es ja auf jeden fall gehen
(cli version is getestet)